hkm/management/commands/clean_unused_data.py

```diff
@@ -22,8 +22,10 @@ def handle(self, *args, **kwargs):
         self.stdout.write("Old data cleaning started! Removing data older than " + str(days) + " days.")
         counted_date = timezone.now() - timedelta(days=days)
 
-        # Find and delete unused users
-        users = User.objects.filter(last_login__lte=counted_date).delete()
+        # Find and delete unused users. Do not delete superusers or staff users.
+        users = User.objects.filter(last_login__lte=counted_date, is_superuser=False, is_staff=False,
+                                    profile__is_museum=False, profile__is_admin=False).delete()
+        print(users)
 
         # Find and delete old TmpImages, Feedbacks and Orders
         temps = TmpImage.delete_old_data(counted_date)
```

Comment:
As discussed, please add filtering to temps/feedbacks/orders so that only anonymous rows (= those which don't have a user_id/creator_id) are removed.

Reply:
`Feedback` was the only one that was missing check for user. It is now added.